commit 032b42c9371bfba869b90d7bd13f2492cadb9065
Author: Volker Krause <vkrause@kde.org>
Date:   Fri Jul 7 16:11:17 2023 +0200

    Adapt to Qt 5.15.10 changes in androiddeployqt
    
    Things still build but without the new way of specifying the NDK version
    binary stripping doesn't work resulting several times larger APKs.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bdc8153..a2cf1ac 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,7 +13,7 @@ project(ktrip VERSION ${RELEASE_SERVICE_VERSION})
 
 set(QT_MIN_VERSION "5.15.0")
 if (ANDROID)
-    set(QT_MIN_VERSION "5.15.8")
+    set(QT_MIN_VERSION "5.15.10")
 endif()
 set(KF_MIN_VERSION "5.88.0")
 
diff --git a/android/build.gradle b/android/build.gradle
index 1eeecb6..d15be5e 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -47,6 +47,7 @@ android {
     compileSdkVersion androidCompileSdkVersion.toInteger()
 
     buildToolsVersion androidBuildToolsVersion
+    ndkVersion androidNdkVersion
 
     sourceSets {
         main {
